pom.xml

```diff
@@ -2585,6 +2578,11 @@
         <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
         <version>${version.org.jboss.spec.javax.xml.bind.jboss-jaxb-api_2.3_spec}</version>
       </dependency>
+      <dependency>
+        <groupId>javax.xml.bind</groupId>
+        <artifactId>jaxb-api</artifactId>
+        <version>${version.javax.xml.bind.jaxb-api}</version>
+      </dependency>
```

Comment:
we should probably decide what jaxb api/impl we need to use as now we have like 3 different bundled jaxb 
jakarta, RI and jboss one for APi

See https://issues.redhat.com/browse/DROOLS-5642 i created the other day as some of deps are not colliding with these duplicated API/impl classes